vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml

```diff
@@ -24,7 +24,7 @@ spec:
   restartPolicy: "Never"
   containers:
     - name: eventshub
-      image: registry.ci.openshift.org/openshift/knative-nightly:knative-eventing-test-eventshub
+      image: {{ .image }}
```

Comment:
We can't do tis!

/hold

Reply:
We MUST wait on: https://github.com/openshift/knative-eventing/pull/1546

and than we should _NOT_ revert this file.

Reply:
let's revert this too